src/bors/handlers/pr_events.rs

```diff
@@ -214,4 +218,24 @@ PR will need to be re-approved."#,
         })
         .await;
     }
+
+    #[sqlx::test]
+    async fn update_base_branch_on_pr_edited(pool: sqlx::PgPool) {
+        run_test(pool, |mut tester| async {
+            tester
+                .edit_pull_request_with_pr(
```

Comment:
What is the default base branch? The test should first assert (or explicitly set) the initial state, then do an edit that  changes it to something explicit and then check that it was changed. From this test it's not obvious what is checked and what was the original state.